core/src/main/resources/error/error-classes.json

```diff
@@ -5627,4 +5627,4 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   }
-}
+}
```

Comment:
Unnecessary change. Please, revert it back.

Reply:
This is done.

Reply:
I have proposed a new PR to address this issue
PR: https://github.com/apache/spark/pull/41256